index.js

```diff
@@ -1,8 +1,67 @@
-// Iteration 1: Names and Input
 console.log("I'm ready!");
 
+// Iteration 1: Names and Input
+
+let Hacker1 = "Erik";
```

Comment:
You should not declare variables with the first letter capitalized (you will see later that, that is used for classes), so it should be `hacker1` if your variable has two words use camelCase `hackerOne`